cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/ProgramWorkflowAction.java

```diff
@@ -60,22 +61,22 @@ public void configure() {
 
   @Override
   public void run() {
+    String prettyProgramType = programType != null ? ProgramType.valueOf(programType.name()).getPrettyName() : "";
```

Comment:
@anwar6953 Doesnt seem like programType can be NULL. Not sure why the existing code was checking for that. DUT ran ok with the change: https://builds.cask.co/browse/CDAP-DUT5698-3